.travis.yml

```diff
@@ -21,8 +21,15 @@ jdk:
   - oraclejdk7
   - oraclejdk8
 
+services:
+  - docker
+
+before_install:
+  - docker pull ibmcom/ibmjava:8-sdk
+
 script:
-  - mvn test apache-rat:check clirr:check checkstyle:check findbugs:check javadoc:javadoc
+  - mvn
+  - docker run -v `pwd`:/work library/ibmjava:8-sdk /bin/bash -c "apt-get update && cd work && apt-get install -y maven && mvn"
```

Comment:
I'm about to pull this in, but I wonder if we could put in either issues or PR's on the official "maven" docker image whose docker files are given here: https://github.com/carlossg/docker-maven such that we could get oracle java as well as ibm java supported.

I've opened the following issue for tracking purposes:
https://github.com/carlossg/docker-maven/issues/25

I'm not sure, but I could see licensing being the main blocker to those being brought into the project.